packages/base-controller/src/BaseControllerV2.ts

```diff
@@ -113,17 +113,6 @@ export class BaseController<
 
   public readonly metadata: StateMetadata<ControllerState>;
 
-  /**
```

Comment:
@Gudahtt I know that we tried to remove this once, and you previously reverted it because we wanted to avoid another major `base-controller` release. What are your thoughts on this?

Reply:
It looks like we could piggy-back on the major release that will follow https://github.com/MetaMask/core/pull/3611.

Reply:
It looks like #3611 is blocked by #3908, which is blocked by the TransactionController multichain PR. So once that's merged and we get a new release of TransactionController and SelectedNetworkController we can return to this.

Reply:
As per our discussion, there should be no issue with merging this PR first and keeping it unreleased, as long as it's eventually batched into the same release as #3611. The contents of this PR and #3908 are not dependent on each other, so there's no need to wait for it to merge first.

Note from @mcmire: Until #3611 is ready for release, we will need to monitor release candidates to make sure `@metamask/composable-controller` is not included.

Reply:
Removed in https://github.com/MetaMask/core/pull/3904/commits/1107e251e8818104ed79e564bd86282d4c522eaf. To be reapplied after https://github.com/MetaMask/core/pull/3611 is merged.

(Changes to base-controller may be disruptive for releases due to https://github.com/MetaMask/create-release-branch/issues/91)